java/src/jmri/LightManager.java

```diff
@@ -204,12 +204,12 @@
     public boolean allowMultipleAdditions(@Nonnull String systemName);
 
     /**
-     * Provide a connection specific tooltip for the Add new item beantable pane.
+     * Provide a connection system specific tooltip for the Add new item beantable pane.
```

Comment:
It's not system-specific. Even within one system, different managers may be enforcing different naming.  The "powerline" system for example has different naming depending on whether the particular connection is through X10 or Insteon.

It's truly a manager-specific tooltip, because it comes from a specific manager. That covers cases like the Proxy managers, which have to figure out how to provide something that's not very specific...